### ISSUE TITLE
map cc values according to a "graph/lookup table"?

### ISSUE THREAD

**Issue by lokkiikkol** [x42/midimap.lv2#2]
hi, just discovered your midi plugins, nice!

i am trying to work with them in element on OS X, and it does not yet work perfectly. lv2 support in element (on osx) is quite new, so that might be the issue as well.

a general question for your midimap plugin,

is it possible to remap a cc message value according to a curve? so i.e. incoming midi cc 8 0 to 127 is used as a lookup value for a table that i created? so one turn of a midi knob could become a sine wave for example. 

the greatest thing would actually be a plugin with a small graph window where you can draw the curve, but i would go with the textfile approach if midimap can do something like that. it was not obvious from the docs, if that is possible.

**Comment by x42** [comment]
> is it possible to remap a cc message value according to a curve? 

No, not directly. you can obviously generate the mapping for each of the 127 values according a curve manually.

In any case this MIDI map plugin has been mostly superseded by https://github.com/OpenMusicKontrollers/moony.lv2

> the greatest thing would actually be a plugin with a small graph window where you can draw the curve

I've seen some of those for velocity curves, but not for CCs. In any case that would be a different plugin unrelated to midimap or midifilters.
